### PR TITLE
Add support to pick latest task definition

### PR DESCRIPTION
**Pull request by stackexpress-tanmaya**
# Description

Fetches the latest task definition and supplies that to ECS service definition to avoid issues in case an external CI/CD system is involved.
Fixes issue https://github.com/umotif-public/terraform-aws-ecs-fargate/issues/40

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -141,6 +141,10 @@ locals {
   ]
 }
 
+data "aws_ecs_task_definition" "task" {
+  task_definition = aws_ecs_task_definition.task.family
+}
+
```
Comment (seanpascual) [inline main.tf:147]: Please move this to data.tf and it should be good

**Review by seanpascual — CHANGES_REQUESTED** [review]
I've done testing with this and it works perfectly.

Please move the data block to data.tf and then I can merge and release.

Thanks

Sean

File: main.tf (hunk 1 above)
Comment (stackexpress-tanmaya) [reply main.tf]: Done. Thanks!

**Review by Ohid25 — APPROVED** [review]
LGTM 👍

**Review by seanpascual — APPROVED** [review]
Looks good

**Pull request merged**